support-frontend/assets/pages/paper-subscription-checkout/components/paperCheckoutForm.jsx

```diff
@@ -103,18 +112,29 @@ function mapStateToProps(state: WithDeliveryCheckoutState) {
     isTestUser: state.page.checkout.isTestUser,
     country: state.common.internationalisation.countryId,
     stripeSetupIntentEndpoint: getGlobal('stripeSetupIntentEndpoint'),
+    csrf: state.page.csrf,
+    currencyId: state.common.internationalisation.currencyId,
+    payPalHasLoaded: state.page.checkout.payPalHasLoaded,
+    amount: getProductPrice(
```

Comment:
Looks like we could pass in a checkout object to this method, unless it's used in other places. Not really important for this PR though.

Reply:
That would be nice, however it has some optional args, so I might leave it for the moment.